packages/fxa-payments-server/.gitignore

```diff
@@ -0,0 +1,6 @@
+# copied l10n files
+/public/fxa-content-server-l10n
+/public/locales/git-head.txt
+/public/locales/**/*.ftl
```

Comment:
Did we or are we going to *move* the FTL files out of payments-server?

There's no en-US FTL here: https://github.com/mozilla/fxa-content-server-l10n/tree/master/locale/en_US/

Reply:
Also, we probably *don't* want to move en-US FTL out of payment-server since we'll want to make edits to that along with other components. There's no extraction of strings with FTL, so we manage those like any other code changes

Reply:
Yeah, we should add an exception for `/public/locales/en-US/*.ftl` here. The "string extraction" step just copies the en-US locale from payments-server over to fxa-content-server-l10n

https://github.com/mozilla/fxa-content-server-l10n/commit/213c9803bae870bcfb1587b2df0b9190f5a1ebc7#diff-b5fabfd5020a3903580439f0be3efa04R87

Reply:
Added.  Thanks for catching that!